### ISSUE TITLE
Beanpolev's greeting audio missing in SRP

### ISSUE THREAD

**Issue by SurDno** [Decane/SRP#27]
When you get to Beanpolev at Clear Sky base, in vanilla game he usually says one of three greeting phrases.

In SRP readme there is a line that you've:

> Worked around Professor Beanpolev's overlapping greeting audio.

However, that workaround actually makes him play no greeting audio at all.

**Comment by Decane** [comment]
A note on terminology: I use the term "work-around" to refer to a solution that eliminates the occurrence of a bug by avoiding the situation that gives rise to it. In contrast, I understand a "fix" as a solution that prevents the bug from occurring even without the said avoidance.

>[The] workaround actually makes him play no greeting audio at all.

One (and only one) of his greeting audios plays - when you open a conversation window with him. The other greeting audio -- the one that is supposed to play in close proximity of him -- never plays because I disabled it to avoid the two overlapping; a disturbingly frequent occurrence in vanilla.

Like #26, this is a considered, albeit non-ideal, outcome - when I devised this work-around in late 2008, I did not have the skills/know-how to actually _fix_ the issue. Since I may be able to do so now, I'm willing to label this a regression and keep it open despite it being deliberate.
